en/TOC.md

```diff
@@ -33,12 +33,13 @@
   - [Monitor TiDB Using Helm](monitor-a-tidb-cluster.md)
   - [Monitor TiDB Using TidbMonitor](monitor-using-tidbmonitor.md)
 + Maintain
-  - [Destroy a TiDB cluster](destroy-a-tidb-cluster.md)
+  - [Destroy a TiDB Cluster](destroy-a-tidb-cluster.md)
   - [Restart a TiDB Cluster](restart-a-tidb-cluster.md)
   - [Maintain a Hosting Kubernetes Node](maintain-a-kubernetes-node.md)
   - [Collect TiDB Logs](collect-tidb-logs.md)
   - [Enable Automatic Failover](use-auto-failover.md)
   - [Enable Admission Controller](enable-admission-webhook.md)
+  - [Use PD Recover to Recover the PD Cluster](pd-recover.md)
```

Comment:
How about using `Restore` for the 2nd `Recover` to avoid two same words? And `restore` is widely used in our docs. If you agree, please check and update all occurrences. Thanks!
```suggestion
  - [Use PD Recover to Restore the PD Cluster](pd-recover.md)
```